<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Session</class>
 <widget class="QWidget" name="Session">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="1">
      <widget class="QLineEdit" name="le_ConnectionType"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>ConnectionType</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>RoamingPoliciy</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="le_RoamingPolicy"/>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="le_AllowedBearers"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>AllowedBearers</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="le_EmergencyCall">
       <property name="inputMask">
        <string>B;0</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="31" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Session Name</string>
       </property>
      </widget>
     </item>
     <item row="31" column="1">
      <widget class="QLineEdit" name="le_SessionName"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_IPv4">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>IPv4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>IPv6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_IPv6">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Bearer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_Bearer">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_Name">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_State">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>SessionMarker</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_SessionMarker">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>InterfaceName</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_Interface">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Priority</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="le_Priority">
       <property name="inputMask">
        <string>B;0</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>AvoidHandover</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="le_AvoidHandover">
       <property name="inputMask">
        <string>B;0</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>StayConnected</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="le_StayConnected">
       <property name="inputMask">
        <string>B;0</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>PeriodicConnect</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="le_PeriodicConnect"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>IdleTimeout</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="le_IdleTimeout"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>EmergencyCall</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_Create">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Destroy">
       <property name="text">
        <string>Destroy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Connect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Disconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_SessionEnable">
       <property name="text">
        <string>Session Enable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_SessionDisable">
       <property name="text">
        <string>Session Disable</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_Priority</tabstop>
  <tabstop>le_AllowedBearers</tabstop>
  <tabstop>le_AvoidHandover</tabstop>
  <tabstop>le_StayConnected</tabstop>
  <tabstop>le_PeriodicConnect</tabstop>
  <tabstop>le_IdleTimeout</tabstop>
  <tabstop>le_EmergencyCall</tabstop>
  <tabstop>le_SessionName</tabstop>
  <tabstop>pb_Create</tabstop>
  <tabstop>pb_Destroy</tabstop>
  <tabstop>pb_Connect</tabstop>
  <tabstop>pb_Disconnect</tabstop>
  <tabstop>pb_SessionEnable</tabstop>
  <tabstop>pb_SessionDisable</tabstop>
  <tabstop>pb_Quit</tabstop>
  <tabstop>le_SessionMarker</tabstop>
  <tabstop>le_Bearer</tabstop>
  <tabstop>le_Name</tabstop>
  <tabstop>le_State</tabstop>
  <tabstop>le_IPv4</tabstop>
  <tabstop>le_IPv6</tabstop>
  <tabstop>le_Interface</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
